dashboard/lib/api/v1/school_autocomplete.rb

```diff
@@ -43,6 +43,23 @@ def self.get_matches(query, limit, use_new_search)
     end
   end
 
+  def self.get_zip_matches(query, limit)
+    limit = format_limit(limit)
+    rows = School.limit(limit)
+    query = "#{query[0, 5]}%"
+    rows = rows.where("zip LIKE ?", query)
+    # For private schools, we don't yet have a way to determine churn/inactive
+    # schools, so we surface them all to teachers.
+    # For public and charter schools, we filter on 'known open' schools as
+    # determined by nces 'status' (see school.rb for OPEN_SCHOOL_STATUSES logic)
+    # to prevent showing duplicate/inactive schools to the user.
+    current_import_year = School.maximum(:last_known_school_year_open)
```

Comment:
not sure what the best option here is but could we hard code this? I'm torn between not wanting to hard code a constant we change every year and this extra query. 🤔

Reply:
I'm surprised to hear this! I think the NCES ingest process is already a big cumbersome so was hoping to not add another step. That said, because I didn't see any performance lag when I updated to this version, I wasn't thinking of performance implications. Should we discuss offline?

Reply:
We spoke offline and decided to cache the result

Reply:
Caching is added:) Resolving!